src/main/kotlin/me/serce/solidity/lang/resolve/ref/SolImportPathReference.kt

```diff
@@ -97,6 +101,34 @@ class SolImportPathReference(element: SolImportPathElement) : SolReferenceBase<S
       }
     }
 
+    private fun remappingsFromFoundryConfigFile(file: VirtualFile): List<Pair<String, String>> {
+      val foundryConfigFile = file.findFileByRelativePath("foundry.toml")
+      val remappingsResult = arrayListOf<Pair<String, String>>()
+      if (foundryConfigFile != null) {
+        val mapper = TomlMapper()
```

Comment:
Let's also initialise the mapper lazily once as they're meant to be reused in Jackson (see [#jackson-performance](https://github.com/fasterxml/jackson-docs/wiki/presentation:-jackson-performance#basics-things-you-should-do-anyway)). You can do this by adding this line in the companion object. 
```kotlin
    val tomlMapper by lazy { TomlMapper() }
```